vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -593,14 +585,8 @@ class HomeActivity :
                     .setPositiveButton(R.string.yes) { _, _ -> bugReporter.openBugReportScreen(this) }
                     .setNegativeButton(R.string.no) { _, _ -> bugReporter.deleteCrashFile() }
                     .show()
-<<<<<<< HEAD
-        } else if (shouldShowDisclaimerDialog(this)) {
-            showDisclaimerDialog(this)
-            homeActivityViewModel.handle(HomeActivityViewActions.DisclaimerDialogShown)
```

Comment:
we should keep this condition which was specific to tchap

Reply:
_shouldShowDisclaimerDialog_ and _showDisclaimerDialog_ are not available any more
Should I get back the code from older version ?